README.md

```diff
@@ -0,0 +1,228 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+https://johntefresh.github.io/HelloMicroverse-Project/
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+A new developer can learn how linter work.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A working laptop, desktop or a phone.
+
+### Setup
+
+Clone this repository to your desired folder:
+  
+  cd newfeature
+  
+  git clone https://github.com/JohnteFresh/HelloMicroverse-Project.git
+
+
+### Install
+
+Install this project with:
+
+  cd newfeature
+  
+  gem install
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+### Deployment
+
+You can deploy this project using:
+
+Your prefered browser.
+
+A working laptop or phone.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors">Johnson Irungu</a>
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/JohnteFresh)
+- Twitter: [@twitterhandle](https://twitter.com/RoadKing254)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/johnson-irungu-85475a268/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[A logo]**
+- [ ] **[My contacts]**
+- [ ] **[Time available]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to contact me if you have somework for me.
+
+Rate my work in Google play store, Udemy, and Follow me on LinkedIn
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the knowledge they have instiled in me and all my codinng partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] Kindly remove this dummy text from your README as part of the customization requirement.